src/Omnipay/Common/Message/AbstractRequest.php

```diff
@@ -476,6 +478,8 @@ public function setTransactionId($value)
     /**
      * Get the transaction reference.
      *
+     * The transaction reference is the identified generated by the payment gateway.
```

Comment:
typo - should be "identifier". May be worth saying "remote" payment gateway, but it I may be seeing ambiguity that isn't there.